### PR TITLE
docs: adapt READMEs

### LINKED ISSUE #8: Update `README`

### PR DESCRIPTION
**Pull request by guidanoli**
Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,73 @@
...
+cd onchain
+yarn
+```
+
+_(Optional)_ If you plan to run the on-chain tests, there still some setup left to do.
+Assuming you are on the `onchain` directory, you may run the following commands.
+
+```sh
+cd rollups
+yarn proofs:setup
```
Comment (ZzzzHui) [inline README.md:35]: should we refer to https://github.com/cartesi/rollups-contracts/tree/feature/adapt-readme/onchain/rollups/test/foundry/dapp/helper here?

Comment (guidanoli) [reply README.md:35]: I think it's better to move the documentation from there to here, actually.
This is a crucial part for running tests, so it's good to have it all in once place.

Comment (ZzzzHui) [reply README.md:35]: Agree

File: README.md [hunk 2]
```diff
@@ -1,24 +1,72 @@
...
+This repository uses [Yarn](https://classic.yarnpkg.com/en/docs/install) to manage npm packages.
+In order to install them, please, run the following commands.
+
+```sh
+cd onchain
+yarn install
+```
+
+_(Optional)_ If you plan to run the on-chain tests, there still some setup left to do.
+Assuming you are on the `onchain` directory, you may run the following commands.
```
[the remark this answers is not in the document]
Comment (guidanoli) [reply README.md:31]: That's true, but that's also kind of implicit when we put Docker in the list of dependencies, right?
Docker would be of no use if the daemon wasn't running on the background already.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,72 @@
...
+
+```sh
+cd rollups
+yarn proofs:setup
+```
+
+Now, you may want to test the on-chain code with [Forge](https://book.getfoundry.sh/getting-started/installation).
```
[the remark this answers is not in the document]
Comment (guidanoli) [reply README.md]: This has been addressed by 14292cb.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (guidanoli) [reply README.md]: This has been addressed by 14292cb.

File: README.md [hunk 4]
```diff
@@ -1,24 +1,74 @@
...
 
 ## Table of contents
 
 - [Documentation](#documentation)
 - [Experimenting](#experimenting)
 - [Talk with us](#talk-with-us)
 - [Contributing](#contributing)
-- [Setting up](#setting-up)
-- [Testing](#testing)
 - [License](#license)
```
Comment (ZzzzHui) [inline README.md:60]: Do you think it looks a bit odd if the table of contents is in the middle of a page? Should we change it back similar to what it was to the beginning of a page?

Comment (guidanoli) [reply README.md:60]: Ok, let me fix that real quick.

Comment (guidanoli) [reply README.md]: This has been addressed by 039c16b.

**Pull request merged**